### PR TITLE
Expand callback section instructions

### PR DESCRIPTION
**Pull request by cocojoe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ react-native link react-native-auth0
 
 #### Android
 
-In the file `android/src/app/AndroidManifest.xml` you must make sure the main activity of the app has launch mode value of `singleTask` and that it has the following intent filter
+In the file `android/app/src/main/AndroidManifest.xml` you must make sure the **MainActivity** of the app has a **launchMode** value of `singleTask` and that it has the following intent filter:
```
Comment (lbalmaceda) [inline README.md]: Path is wrong image

Comment (cocojoe) [reply README.md]: That is the path inside the react-native-auth0 module. Not the path for the users project

Comment (cocojoe) [reply README.md]: screen shot 2017-10-04 at 15 14 56

Comment (cocojoe) [reply README.md]: The doc is in the context of configuring the users react-native project. Just as the `ios` one is.

**Pull request merged**